src/index.html

```diff
@@ -10,7 +10,30 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body>      
+    <form class="search-bar" data-qa="big"> 
+      <input 
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="search-bar__input search-bar__input--big"
+        data-qa="keypress"
+        >
+      <img 
+        src="images/Search.svg" 
+        alt="search icon" 
+        class="search-bar__icon--big"
+      >
+    </form>
+
+    <form class="search-bar" data-qa="small"> 
+      <input 
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="search-bar__input search-bar__input--small"
+        >
+      <img 
+        src="images/Search.svg" 
+        alt="search icon" 
+        class="search-bar__icon--small"
```

Comment:
Modifier always goes with block/element
Modifier applies only difference between this current element `<img>` and BEM block `search-bar__icon`
```suggestion
        class="search-bar__icon search-bar__icon--small"
```